server/service/src/sensor/berlinger.rs

```diff
@@ -1,3 +1,4 @@
+use anyhow::Context;
```

Comment:
I did some refactoring in this file, best viewed with white space diff off:
* Return type of ReadSensor (number of logs and breaches)
* Add ReadSensorError (makes it easier to cast and return error in main method)
* Read sensor method processes just one sensor